### DISCUSSION TITLE
Add a way to listen for events on root component

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/rfcs#699] [General]
Right now you can put the prop into the root component which is awesome.

```typescript
var app = createApp(rootCOmponent, { modelValue: props.modelValue });
```

But it would be great if we could do:

```typescript
app.on("update:modelValue", function(newValue) {
    // ...
})
```

I'm using this to render vue into an iframe, so I need to get the updates and pass them back to my main application

**Comment by posva** [comment]
Using an intermediate component for the events already allows this
